chapters/ch05/1.open-information-extraction.ipynb

```diff
@@ -10,21 +10,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
```

Comment:
I'm guessing these warnings are likely due to the `visiontorch` library now being imported without a version in the requirements.txt (so it's always pulling the newest version torch/pytorch). Possibly when the version of torchvision is set is should be a slightly older version for compatibility with whatever is calling this?

Reply:
Can obviously live with some warning if they are preexisting; just might be an opportunity to squash them by choosing a version of visiontorch / torch that is compatible with whatever is throwing this sderr message.

Reply:
At some point in the future if you want we can spend more time fiddling with dependency versions to eliminate warnings.